src/backend/core/api/client/viewsets.py

```diff
@@ -267,6 +267,8 @@ def get_queryset(self):
             queryset = queryset.filter(
                 is_active=True,
             )
+            if organization := self.request.user.organization:
```

Comment:
You can remove a query by using the 'user.organization_id' then filter with pk.
This way django won't make a query to fetch all organization's data.

Reply:
works like a charm but, to lower the number of queries I have to add this filter to the active user filter.
Can I safely remove the "if" on line 270 ? Do all users have one an only one organization ? This is not the case in prod but we can fix these discrepancies